### ISSUE TITLE
Separate Surface from Volumetric Pipelines

### ISSUE THREAD

**Issue by nimzodisaster** [pBFSLab/DeepPrep#183]
It is useful having both types of outputs, but some people may wish to only do the volumetric pipeline, not the surface pipeline.
It would be nice to have a flag that will only run that much shorter pipeline, which setting --bold_surface_spaces 'NONE' does not seem to do.

**Comment by NingAnMe** [comment]
I completely agree with this suggestion; it could save a lot of resources. I'll add it to the plan and prioritize this task.

**Comment by nimzodisaster** [comment]
I've been working on it, and just got it to work.
Im attaching the edited files.

The general strategy was to check if --bold_surface_spaces None was given. If set to None, we set some conditional statements in the the nextflow workflows, setting some output value to Channel.empty(), and so on. It may be a bit hacky, but it finished.

[bold_preprocess.py.txt](https://github.com/user-attachments/files/18121700/bold_preprocess.py.txt)
[deepprep.nf.txt](https://github.com/user-attachments/files/18121701/deepprep.nf.txt)


I wonder, do you think synthmorph could do a good job registering the EPI to the T1w?

**Comment by Ireneyou33** [comment]
We attempted to register EPI data in native space to T1w in atlas space, but the results were not ideal. Instead, we applied `bbregister` to align the EPI data in native space with T1w in native space, and then applied `synthmorph` to register the T1w data to MNI152.
